macosfrontend/macosfrontend.cpp

```diff
@@ -205,6 +224,23 @@ bool MacosFrontend::keyEvent(ICUUID uuid, const Key &key, bool isRelease) {
     }
     KeyEvent keyEvent(ic, key, isRelease);
     ic->keyEvent(keyEvent);
+
+    if (simulateKeyRelease_ && !isRelease && !key.isModifier()) {
```

Comment:
In vscodium I press cmd+n to open a new tab, but with combo pinyin the key release triggers a `er`.

Reply:
Seems like an upstream problem. `chord_processor` returns handled for super+anything. Created a PR: https://github.com/rime/librime/pull/808